src/libAtomVM/otp_ssl.c

```diff
@@ -42,6 +42,8 @@
 // #define ENABLE_TRACE
 #include <trace.h>
 
+#define TAG "otp_ssl"
```

Comment:
Is this a leftover?

Reply:
I fixed a warning. TAG was (wrongly) defined in otp_platform_socket.h.